### PR TITLE
#110: Replace yaml loader with one that supports env value substitutions.

### PR DESCRIPTION
**Pull request by archfz**

### CODE REVIEW

**Comment by archfz** [comment]
This is my first change in python language, so advise if anything seems off.

**Review by jertel — CHANGES_REQUESTED** [review]

File: Makefile [hunk 1]
```diff
@@ -21,10 +21,13 @@ test-elasticsearch:
...
-	docker-compose --project-name elastalert run --rm tox
+	docker-compose --project-name elastalert run --rm tox tox -- $(filter-out $@,$(MAKECMDGOALS))
 
 clean:
 	make -C docs clean
 	find . -name '*.pyc' -delete
 	find . -name '__pycache__' -delete
 	rm -rf virtualenv_run .tox .coverage *.egg-info build
+
+%:
```
[the remark this answers is not in the document]
Comment (archfz) [reply Makefile:32]: This is required unfortunately to be able to pass arguments to make. But i think it's a small price to pay for a good feature. The only thing is that commands that do not exist won't print an error.

File: elastalert/alerts.py [hunk 2]
```diff
@@ -26,7 +26,6 @@
 from jira.exceptions import JIRAError
 from requests.auth import HTTPProxyAuth
 from requests.exceptions import RequestException
-from staticconf.loader import yaml_loader
```
[the remark this answers is not in the document]
Comment (archfz) [reply elastalert/alerts.py:29]: I think we can remove.

[the remark this answers is not in the document]
Comment (jertel) [reply elastalert/alerts.py:29]: @nsano-rururu Do you concur? You've spent a lot of time reviewing those dependencies so your opinion is appreciated.

[the remark this answers is not in the document]
Comment (nsano-rururu) [reply elastalert/alerts.py:29]: @jertel  Yes. is consent.

[the remark this answers is not in the document]
Comment (archfz) [reply elastalert/alerts.py:29]: @jertel removed

**Review by jertel — APPROVED** [review]

**Pull request merged**